ksql-engine/src/main/java/io/confluent/ksql/function/udaf/max/IntegerMaxKudaf.java

```diff
@@ -0,0 +1,61 @@
+/**
```

Comment:
nit: You should update your copyright template. The copyright is not a Javadoc, so should not start with `/**`. It just generates warnings in most IDEs.